### ISSUE TITLE
Installed refind but not showing in boot options

### ISSUE THREAD

**Issue by nex86** [jlobue10/rEFInd_GUI#4]
I used the GUI to install refind.
it did create the files in /BOOT/EFI/REFIND.

But now how do I use it? 
it does not show up in my boot options.

using bazzite btw

**Comment by UnaTried** [comment]
You can use efibootmgr, it is very straightforward. An example command would be 
`sudo efibootmgr -c -L rEFInd -d /dev/sda2 -l /EFI/REFIND/refindx64.efi`

-c creates the boot entry
-L sets label (name), in this case rEFInd
-d sets the device the refind efi file is on (/BOOT is a mount) for it so do lsblk and look for /BOOT under`MOUNTPOINTS` and look at the name on the same line
-l sets the loader file for x64 it would be the refind folder (/EFI/REFIND/), with refindx64.efi inside of it. You don't need to add /BOOT because it is a mountpoint in LINUX not your system and rEFInd your BIOS/system know how to get the correct file. if you do `efibootmgr` without any sudo or arguments it will show you that it translates your device (/dev/sda2) into a BIOS/system readable string

**Comment by nex86** [comment]
Thanks I tried that, still doesn't work.
The entry is in efibootmgr but not present in the boot loader list (F12)

Motherboard is a Gigabyte b550 aorus elite v2, maybe incompatiblity issues?

**Comment by jlobue10** [comment]
I think the issue is that the OS filesystem locks and protections that are in place on Bazzite block my intended installation script from performing certain steps. So this script which works without issue for me on Nobara, has issues with Bazzite.

The workaround that I think DeckWizard had found and used is to actually just manually run the `$HOME/Downloads/refind-bin-0.14.2/refind-install` procedure (from that location or wherever it was downloaded to for you). The GUI is then meant to make customization of icons and backgrounds much easier than command line.

**Comment by nex86** [comment]
ok making progress, now its showing up in the boot menu
but I'm being greated by a giant red screen about the bootloader not being signed.
would there be a disadvantage with disabling secureboot at this point or is there a way to get to working with secureboot?

**Comment by nex86** [comment]
I tried the steps under "Secure boot considerations"

But I'm unable to install this sbctl since the package is not available on bazzite

**Comment by jlobue10** [comment]
Yes I'm not familiar with all of Bazzite's `ujust` shortcuts and aliases. Normal `copr` transactions (setup and installation) is likely different enough that it may just be worth finding out how to sign the `/boot/efi/EFI/refind/refind_x64.efi` binary with Bazzite's recommended secure boot signing ways. I'd ask in their discord personally, if I were you.

**Comment by nex86** [comment]
> Yes, I'm not familiar with all of Bazzite's `ujust` shortcuts and aliases. Normal `copr` transactions (setup and installation) are likely different enough that it may just be worth finding out how to sign the `/boot/efi/EFI/refind/refind_x64.efi` binary with Bazzite's recommended secure boot signing ways. I'd ask in their discord personally, if I were you.

they just told me to disable secureboot entirely which I did.
it works but sometimes refind just freezes, I can't navigate in the menus and I have to reboot the Ally.

**Comment by jlobue10** [comment]
Fair enough. Can you show me what your `/boot/efi/EFI/refind/refind.conf` file looks like, and also the output of `efibootmgr`?

**Comment by Skyyblaze** [comment]
I have the same issue installing it from Windows 11 with Bazzite as my other dualboot option. rEFInd gets installed and is present on the EFI Partition but it doesn't show up in the BIOS or boot-selector of my ROG Ally.
